chart/README.md

```diff
@@ -541,14 +541,62 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### HPA (Horizontal Pod Autoscaling)
```

Comment:
I would just title this as `Autoscaling`

Reply:
commit; e3ed1e84384a7cf01707582318521ed9f87e7195